tools/changelogger-release.sh

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+BASE=$(cd $(dirname "${BASH_SOURCE[0]}")/.. && pwd)
+. "$BASE/tools/includes/check-osx-bash-version.sh"
+. "$BASE/tools/includes/chalk-lite.sh"
+
+# Print help and exit.
+function usage {
+	cat <<-EOH
+		usage: $0 [-v] [-b] <slug>
+
+		Prepare a release of the specified project and everything it depends on.
+		 - Run \`changelogger write\`
+		 - Run \`tools/project-version.sh\`
+
+		Pass \`-b\` to release pass \`--prerelease=beta\`.
```

Comment:
That wasn't super clear to me. Maybe we could be more verbose in those explanations?

Reply:
Sure! Any suggestions?

I think `-b` may be the answer to your question earlier about "we end up updating jetpack.php to 9.6 for example, even though we only want to release 9.6-beta first".

Reply:
How about 

```suggestion
		Pass \`-b\` to pass the \`--prerelease=beta\` parameter and prepare a Beta release.
```

Reply:
Hmm. Not bad, but how about
```suggestion
		Pass \`-b\` to prepare a beta release by passing \`--prerelease=beta\` to changelogger.
```